### PR TITLE
feat(root): Fix JS build and introduce playground applications

### PR DESCRIPTION
**Pull request by desiprisg**
### What changed? Why was the change needed?

I've attempted to fix the build in this PR. I've also worked around our css problem for now with a file rename. The `index.directcss` file shouldn't really change often, if not ever. With a .css suffix, the inline import plugin doesn't seem to work as expected.
For now, I've used the plugin to inject the css in HEAD on our `<Renderer />` .
Note: In the playground app, I'm lazy loading `@novu/js` as the solid code can't run on the server. This is something the `@novu/nextjs` package will be doing when we introduce it, but I'm doing in right now to allow us to use that app to test the package instead of web.
### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: packages/js/package.json [hunk 1]
```diff
@@ -33,7 +37,7 @@
   "private": true,
   "scripts": {
     "start": "pnpm run build -- --watch --sourcemap",
-    "build": "tsup && pnpm run build:umd && pnpm run post:build",
```
Comment (BiswaViraj) [inline packages/js/package.json:36]: ❓ **question:** can we revert this change?‏

File: packages/js/package.json [hunk 2]
```diff
@@ -16,12 +16,16 @@
...
+      "require": "./dist/cjs/ui/index.cjs",
+      "import": "./dist/esm/ui/index.mjs",
+      "default": "./dist/esm/ui/index.mjs"
     },
-    "./ui/index.css": "./dist/ui/index.css"
+    "./ui/index.css": {
+      "require": "./dist/cjs/ui/index.css",
+      "import": "./dist/esm/ui/index.css",
+      "default": "./dist/esm/ui/index.css"
+    }
```
Comment (BiswaViraj) [inline packages/js/package.json]: ☑ **chore:** This can be removed as you're using .directcss now‏

File: playground/nextjs/.eslintignore [hunk 3]
```diff
@@ -0,0 +1 @@
+/
```
Comment (BiswaViraj) [inline playground/nextjs/.eslintignore:1]:
```suggestion

```

❓ **question:** Maybe a typo‏

File: packages/js/src/ui/novuUI.tsx [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
 import { Appearance } from './context';
-import './index.css';
 import { generateRandomString } from './helpers';
 import { Localization } from './context/LocalizationContext';
-import { NovuComponent, NovuComponentControls, NovuComponentName, Renderer } from './components/Renderer';
+import { NovuComponent, NovuComponentName, Renderer } from './components/Renderer';
 import { NovuProviderProps } from './types';
 import { createSignal } from 'solid-js';
+//@ts-expect-error inline import esbuild syntax
+import css from 'directcss:./index.directcss';
```
Comment (BiswaViraj) [inline packages/js/src/ui/novuUI.tsx:10]: 🗒 **note (non-blocking):** Not a fan of this hacky approach. But it's fine for now. We should revisit this later for a better solution.
‏

**Review by BiswaViraj — APPROVED** [review]
Great work on this

left a couple of comments

File: playground/nextjs/.eslintignore (hunk 3 above)
Comment (desiprisg) [reply playground/nextjs/.eslintignore:1]: No this is just to ignore everything, as the root one didn't work.

File: packages/js/src/ui/novuUI.tsx (hunk 4 above)
Comment (desiprisg) [reply packages/js/src/ui/novuUI.tsx:10]: We either will use a CDN for this or find another solution altogether. This is to unblock us for now.

File: packages/js/package.json (hunk 1 above)
Comment (desiprisg) [reply packages/js/package.json:36]: I'm letting tsup generate UMD now. We can revisit this though. Didn't touch the webpack config at all.
